Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/README.md

```diff
@@ -738,20 +733,91 @@ Lists details about all member accounts for the current Security Hub master acco
 | AWS-SecurityHub.Members | Unknown | Member details returned by the operation. | 
 | AWS-SecurityHub.NextToken | string | The token that is required for pagination. | 
 
-
-##### Command Example
+#### Command example
 ```!aws-securityhub-list-members```
-
-##### Context Example
-```
+#### Context Example
+```json
 {
     "AWS-SecurityHub": {
         "Members": []
     }
 }
 ```
 
-##### Human Readable Output
-### AWS SecurityHub ListMembers
-**No entries.**
+#### Human Readable Output
+
+>### AWS SecurityHub ListMembers
+>**No entries.**
+
+
+### get-remote-data
+***
+Get remote data from a remote incident. This method does not update the current incident, and should be used for debugging purposes only.
+
+
+#### Base Command
+
+`get-remote-data`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| id | The remote incident ID. | Required | 
+| lastUpdate | The UTC timestamp in seconds of the last update. The incident is only updated if it was modified after the last update time. Default is 0. | Optional | 
```

Comment:
```suggestion
| lastUpdate | The UTC timestamp in seconds since the last update. The incident is only updated if it was modified after the last update time. Default is 0. | Optional | 
```